README.md

```diff
@@ -5,7 +5,7 @@
 [![Lifecycle: experimental](https://img.shields.io/badge/lifecycle-experimental-orange.svg)](https://lifecycle.r-lib.org/articles/stages.html#experimental)
 [![R-CMD-check](https://github.com/RMI-PACTA/pacta.multi.loanbook/actions/workflows/R-CMD-check.yaml/badge.svg)](https://github.com/RMI-PACTA/pacta.multi.loanbook/actions/workflows/R-CMD-check.yaml)
 [![CRAN
-status](https://www.r-pkg.org/badges/version/pacta.multi.loanbook)](https://CRAN.R-project.org/package=pacta.multi.loanbook)
+status](https://www.r-pkg.org/badges/version/pacta.multi.loanbook)](https://CRAN.R-project.org/package=pacta.multi.loanbook){.pkgdown-release}
```

Comment:
This seems to not render correctly:
<img width="704" alt="Screenshot 2024-11-13 at 19 19 24" src="https://github.com/user-attachments/assets/40aa057a-5808-49ee-917b-c98e72c4681a">

Reply:
@cjyetman pretty sure this needs to be a `README.Rmd` for that to work
😂 your favourite!!

Reply:
> @cjyetman pretty sure this needs to be a `README.Rmd` for that to work 😂 your favourite!!

hmmm, you might be right... it works on the main page of the pkgdown site, but maybe/probably will be ignored on GitHub's rendering of the README.md

I'll just comment fence it for now

Reply:
commented out